charts/datafold/values.yaml

```diff
@@ -82,7 +82,7 @@ nginx:
   install: true
 
 config:
-  webWorkers: "5"
+  webWorkers: "4"
```

Comment:
if replicacount is 2, this probably should be 2.

Reply:
changed as per discussion to 1